include/seqan3/core/add_enum_bitwise_operators.hpp

```diff
@@ -18,6 +18,78 @@
 
 namespace seqan3
 {
+/*!\interface seqan3::enum_bitwise_operators
+ * \brief You can expect these functions on all types that implement seqan3::add_enum_bitwise_operators.
+ */
+/*!\name Requirements for seqan3::enum_bitwise_operators
+ * \relates seqan3::enum_bitwise_operators
+ * \brief You can expect these member functions.
+ * \{
+ * \fn operator&(lhs_t lhs, rhs_t rhs)
+ * \brief Returns the binary `&` operator of lhs and rhs.
+ * \param lhs First enum.
+ * \param rhs Second enum.
+ *
+ * \details
+ * \attention This is a concept requirement, not an actual function (however types satisfying this concept
+ * will provide an implementation).
+ *
+ * \fn operator|(lhs_t lhs, rhs_t rhs)
+ * \brief Returns the binary `|` operator of lhs and rhs.
+ * \param lhs First enum.
+ * \param rhs Second enum.
+ *
+ * \details
+ * \attention This is a concept requirement, not an actual function (however types satisfying this concept
+ * will provide an implementation).
```

Comment:
This is essentially copied for each function documentation here and it therefore seems crowded. Consider whether it appears better in the heading, i.e. move it below the \name tag (somewhere after line 24).

Reply:
I think it is better to have it for every function, in case a user is just looking for the specifics of one operator the user might miss an attention that is somewhere above.

Reply:
Sure, fine for me!